src/integration-test/tests/citizen/breathing-space/breathingspace_test.ts

```diff
@@ -0,0 +1,42 @@
+import { PartyType } from 'integration-test/data/party-type'
+import { InterestType } from 'integration-test/data/interest-type'
+import { createClaimData } from 'integration-test/data/test-data'
+import { BreathingSpaceSteps } from 'integration-test/tests/citizen/breathing-space/steps/breathingspace'
+import I = CodeceptJS.I
+
+import { UserSteps } from 'integration-test/tests/citizen/home/steps/user'
+
+const userSteps: UserSteps = new UserSteps()
+const breathingSpaceSteps: BreathingSpaceSteps = new BreathingSpaceSteps()
+let email
+let claimantType
+let defendantType
+let claimData
+let claimRef
+
+Feature('citizen frontend claimant journey for Breathing space')
+
+Before(async (I: I) => {
+  email = await I.getClaimantEmail()
+  claimantType = PartyType.INDIVIDUAL
+  defendantType = PartyType.ORGANISATION
+  claimData = await createClaimData(I, claimantType, defendantType, true, InterestType.NO_INTEREST)
+  claimRef = await I.createClaim(claimData, email)
+
+})
+
+Scenario('Claimant can enter and lift the Breathing space @smoke @nightly', { retries: 3 }, async (I: I) => {
```

Comment:
It must be @smoke-test
```suggestion
Scenario('Claimant can enter and lift the Breathing space @smoke-test @nightly', { retries: 3 }, async (I: I) => {
```